samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -67,7 +67,11 @@ public static MauiApp CreateMauiApp()
 								.UseMauiCommunityToolkitMarkup()
 								.UseMauiCommunityToolkitCamera()
 								.UseMauiCommunityToolkitMediaElement()
+
+#if WINDOWS
```

Comment:
it looks not correct. we need to call UseMauiMaps() if not Windows

Reply:
Also what will happen if a user doesn't supply a valid key? If it results in an exception I don't think think should be the default behavior, the number of users interested in using maps will be a subset and it would be annoying if it throws an exception

Reply:
it won't throw an exception but displays an error message in the middle of the page, but Maps is usable. the exception is thrown only if the key is null or empty

Reply:
I thought about adding the directives for Android and IOS.

This would also require modifying the Manifest file for Android as well as the IOS implementation.

Lastly I would change the title of the menu items and pages to remove the word "Windows"

I decided against it since the The Community Toolkit only provides the Windows implementation.

It felt like I was changing the purpose of the sample.